types/react-text-marquee/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for react-text-marquee v1.0.1
+// Project: github.com/jcgertig/react-text-marquee
+// Definitions by: Boris <https://github.com/captain328>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+declare module "react-text-marquee" {
```

Comment:
The module definition is not needed.. Everything inside this definition can just be outside the module scope..